gateway/api/v1/gpu-jobs.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Just to double check: @psschwei . This approach would be very similar to this, right? 

https://github.com/Qiskit/qiskit-serverless/pull/1443#discussion_r1711688440

Reply:
Yes